### DISCUSSION TITLE
The vercel app is not responding intermittently.

### DISCUSSION THREAD

**Discussion by SwimmingHwang** [vercel/community#2482] [Help] [answered]
### Summary

The vercel app is not responding intermittently.
It works well in secret mode

### Example

zep-production.vercel.app

**Comment by SwimmingHwang** [comment]
It suddenly started not working today

**Comment by newskys** [comment]
Me too.
It seems to be shown in Korea.

**Comment by juicyjusung** [comment]
Same here, in korea

**Comment by qus1225** [comment]
Same here, in korea

**Comment by ssut** [comment]
It seems some Vercel instances are having connectivity issue. The same domain (vercel.com in here) can be accessible at the different IP addresses Vercel DNS respond.

- https://github.com/vercel/community/assets/2366668/c13200a8-2f46-4966-87b8-4a3faa895956
- https://github.com/vercel/community/assets/2366668/48c82b48-899f-4789-a168-8fea9c98ca46

**Comment by hayluna** [comment]
We're experiencing same issue, too. (Korea, in production)
We've been monitoring https://www.vercel-status.com/, but it says all systems are operational.
Situation is getting severe since it started this morning.

**Comment by smaeda-ks** [comment]
We're looking into this.

**Reply by smaeda-ks** [reply]
Something to note, those Vercel IPs you see are served by our upstream provider (AWS) (more details [here](https://vercel.com/blog/behind-the-scenes-of-vercels-infrastructure))

We suspect that some widespread network issues might be going on in Korea right now, affecting the network route from the Korea region to AWS's Global Network. We're checking this with our upstream provider. Will update here.

**Reply by Sierra0318** [reply]
When will it end?

**Reply by Sierra0318** [reply]
It's making me crazy...

**Reply by hovelopin** [reply]
Is there anything we need to do?

**Reply by smaeda-ks** [reply]
We will deliver updates as soon as we have an update to share.
At this point, the issue seems very specific to this particular region (Korea), and if it's caused by local providers/transit networks in that region, it dones't necessarily be our issue. Regardless, we will be sure to keep you updated.

**Reply by smaeda-ks** [reply]
The upstream provider is currently investigating this.

**Reply by smaeda-ks** [reply]
meanwhile, we also posted this event on our statuspage: https://www.vercel-status.com/incidents/pvy58sglqdpl

**Reply by smaeda-ks** [reply]
> Is this caused by AWS or the networks in Korea?

We don't know yet.

**Reply by smaeda-ks** [reply]
Hi all, @SwimmingHwang @newskys @juicyjusung @qus1225 @ssut @hayluna @hovelopin @Sierra0318 @ernybugi @yelin1004 @wjaykim 

Could you test again and see if you're seeing any improvements right now? Let us know your results. We've changed network-related configuration on our side. We can share more details once we can confirm if it's fixed the issue. If not, we will continue to investigate this.

**Reply by hovelopin** [reply]
@smaeda-ks After the improvements, our project works great! However, it was intermittently disconnected so we will keep checking. Thank you for your efforts

**Reply by SwimmingHwang** [reply]
@smaeda-ks The vercel app connect error that occurred has never occurred since 09:21 UTC. 
Thank you for your help 👍

**Reply by smaeda-ks** [reply, marked as answer]
Following up on this. After some more investigation with our engineering team, we found the root cause on our side that was causing this intermittent connection reset in the ICN1 region. We sincerely apologize for the trouble.

We're working on a permanent fix to ensure the same issue will not happen again in the future. We're also adding more granular monitoring/alerting around this failure scenario so that we can catch a similar event sooner next time.

**Comment by ernybugi** [comment]
same here in Korea.

https://spartacodingclub.kr/

**Comment by yelin1004** [comment]
Same here. Is this caused by AWS or the networks in Korea? The issue seems very serious.

**Comment by wjaykim** [comment]
Slack is down, too. It seems to be widespread network issue in Korea.
